API/Controllers/ProjectController.cs

```diff
@@ -792,6 +790,245 @@ public async Task<IActionResult> UnlikeProject(int projectId)
             return Ok(mapper.Map<ProjectLike, UserProjectLikeResourceResult>(projectLike));
         }
 
+
+
+        /// <summary>
+        /// Updates the institutionprivate property of a project to the given value in the body
+        /// and adds the institution of the creator to the project if the institution isn't yet linked
+        /// to the project and institutePrivate is true.
+        /// </summary>
+        /// <param name="projectId">The project identifier</param>
+        /// <param name="institutePrivate">The new value for InstitutePrivate in the given project</param>
+        /// <returns>
+        ///     StatusCode 200 If success,
+        ///     StatusCode 404 If the project or current user couldn't be found.
+        ///     StatusCode 404 The creator of the project doesn't belong to an institution 
+        ///     StatusCode 403 If the user doesn't have the rights to make the project private.
+        /// </returns>
```

Comment:
Try to use these response comments because Swagger uses it for documentation.
https://github.com/DigitalExcellence/dex-backend/blob/develop/API/Controllers/CallToActionOptionController.cs#L89